storage/innobase/fts/fts0fts.cc

```diff
@@ -4835,17 +4837,23 @@ or greater than fts_max_token_size.
 @param[in]	stopwords	stopwords rb tree
 @param[in]	is_ngram	is ngram parser
 @param[in]	cs		token charset
+@param[in]	skip		true if the check should be skipped
```

Comment:
This function is called for non ngram tokens too. What happens there if the variable is set?